doc/ubuntu_release_process.md

```diff
@@ -120,14 +120,13 @@ This is generally not the mechanism that is preferred for any release supported
 #### Cherry-pick Process
 The tool for doing this is in ``uss-tableflip/scripts/cherry-pick``.  It takes as import a commit-ish that it will create a cherry-pick from.
 
-    ## just tag current so you have a revert point.
-    $ git tag --delete xx-current >/dev/null 2>&1; git tag xx-current
-    $ git checkout ubuntu/xenial
+    $ git checkout origin/ubuntu/xenial -b ubuntu/xenial
```

Comment:
I saw some discussion on origin vs upstream;and it seems to me that the previous line (125) was non-judging and might be better.